week 05/netflix-OOP-student/classes/User.class.php

```diff
@@ -0,0 +1,102 @@
+<?php
+	require_once("Security.class.php");

```

Comment:
Hier zou ik liever met een `trait` werken om je klasse niet vast te hangen aan de Security class. 
vb: https://www.sitepoint.com/using-traits-in-php-5-4/